solidity/contracts/middleware/InterchainAccountRouter.sol

```diff
@@ -46,6 +46,17 @@ contract InterchainAccountRouter is Router, IInterchainAccountRouter {
         );
     }
 
+    function initialize(address _mailbox, address _interchainGasPaymaster)
```

Comment:
Do we need this equivalent change for `InterchainQueryRouter`?

Reply:
Yes we do, i thought the tests would cover it but it turns out by default we do pass an ISM even if it is just the default one